AppInsightsWithWebJob/Program.cs

```diff
@@ -37,14 +38,13 @@ static async Task Main(string[] args)
             var host = builder.Build();
             var cancellationToken = new WebJobsShutdownWatcher().Token;
 
-            var telemetryClient = host.Services.GetService<TelemetryClient>();
```

Comment:
You can log exceptions in main if RunAsync throws

Reply:
So a simple `try/catch`? OK, I'll give that a shot.